code/gradle.properties

```diff
@@ -13,8 +13,8 @@
 org.gradle.jvmargs=-Xmx2048m
 
 moduleName=edgeidentity
-moduleVersion=3.0.0
-mavenCoreVersion=3.0.0
+moduleVersion=3.0.0-beta.1
+mavenCoreVersion=3.0.0-beta.1
```

Comment:
Do you want to change this to 
mavenCoreVersion=3.0.0-beta.1-SNAPSHOT

and keep the 
addCoreDependency(mavenCoreVersion)
and   implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion") in edgeidentity build.gradle?